.github/workflows/release.yml

```diff
@@ -106,3 +106,19 @@ jobs:
           tag-name: ${{ env.TAG }}
         env:
           COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
+
+  winget-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get the latest tag
```

Comment:
It's not necessary to calculate this, the `version` field automatically gets the tag and strips the `v` prefix